roles/kafka_connect/tasks/deploy_connectors.yml

```diff
@@ -1,26 +1,62 @@
 ---
+- name: Get Authorization Token for superuser
+  import_role:
+    name: common
+    tasks_from: get_authorization_token.yml
+  vars:
+    oauth: "{{ oauth_enabled }}"
+    oauth_user: "{{ oauth_client_id }}"
+    oauth_password: "{{ oauth_client_password }}"
+    ldap_user: "{{ mds_super_user }}"
+    ldap_password: "{{ mds_super_user_password }}"
+  when: rbac_enabled or kafka_connect_oauth_enabled
+
 - name: Register Kafka Connect Subgroups
   set_fact:
     subgroups: "{{ ((subgroups | default([])) + hostvars[item].group_names) | difference('kafka_broker, ksql, kafka_connect, kafka_rest, kerberos, ksql, schema_registry, zookeeper, kafka_broker_parallel, ksql_parallel, kafka_connect_parallel, kafka_rest_parallel, kerberos_parallel, ksql_parallel, schema_registry_parallel, zookeeper_parallel, kafka_controller, kafka_controller_parallel') }}"
   with_items: "{{groups['kafka_connect']}}"
 
-- name: Add Role Bindings for Connect
+- name: Add Role Bindings for Connect OAuth
+  include_tasks: rbac_connectors.yml
+  vars:
+    connect_user: "{{kafka_connect_oauth_principal}}"
+  when:
+    - rbac_enabled|bool
```

Comment:
Have we tested on non RBAC OAuth cluster with connector?

Reply:
Yes plain-rhel scenario does that.